From 5821ba6d4cead11aaf54449660305cb320842a75 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 6 Nov 2018 15:11:53 +0100
Subject: [PATCH] [SQUSH][DNM] Merge tag 'android-9.0.0_r16' into lineage-16.0

commit 19dfcac7421c0c91433a48e20eebf681e2c255f8
Author: Bill Yi <byi@google.com>
Date:   Thu Jul 5 00:55:02 2018 -0700

    Import translations. DO NOT MERGE

    Auto-generated-cl: translation import

    Bug: 64712476
    Change-Id: I31180ea0be44bf409b1a6e248debec4cf553fcf8

Change-Id: Ic783224e1ba3aace81248e0ffdc85fbe1d8a1094
---
 res/values-sw/strings.xml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/res/values-sw/strings.xml b/res/values-sw/strings.xml
index 8fac74b..765a19d 100644
--- a/res/values-sw/strings.xml
+++ b/res/values-sw/strings.xml
@@ -22,7 +22,7 @@
     <string name="description_delete_button" msgid="88733669653753441">"futa"</string>
     <string name="description_image_button_plus" msgid="7821061347207448362">"jumlisha"</string>
     <string name="description_voicemail_button" msgid="6281470684653439120">"ujumbe wa sauti"</string>
-    <string name="default_notification_description" msgid="6011631627871110785">"Sauti chaguo msingi (<xliff:g id="DEFAULT_SOUND_TITLE">%1$s</xliff:g>)"</string>
+    <string name="default_notification_description" msgid="6011631627871110785">"Sauti chaguomsingi (<xliff:g id="DEFAULT_SOUND_TITLE">%1$s</xliff:g>)"</string>
     <string name="callFailed_userBusy" msgid="5163219086553610486">"Simu inatumika"</string>
     <string name="callFailed_congestion" msgid="3645995103907418337">"Mtandao unatumika"</string>
     <string name="callFailed_timedOut" msgid="7772951976278867294">"Hakuna jibu, muda umekwisha"</string>
-- 
2.17.1

